packages/block-library/src/post-template/style.scss

```diff
@@ -19,24 +19,18 @@
 		flex-direction: row;
 		display: flex;
 		flex-wrap: wrap;
-
+		gap: 1.25em;
```

Comment:
If we're going to switch to using `gap`, like @aaronrobertshaw mentions it might be worth allowing usage of the block gap CSS variable so that we can give users control over the gap between elements. [Here's an example](https://github.com/wordpress/gutenberg/blob/b7507146ddc18ed8aad0d76d21f431d852f592b3/packages/block-library/src/navigation/style.scss#L279) of a CSS rule we can use from the Navigation block.

Reply:
Yes, if of course, this what i thought about too, but i though i could do it in another PR.